examples/burgers_n2n/config.py

```diff
@@ -0,0 +1,38 @@
+from models.burgers_n2n import *
```

Comment:
If this file is identical to `examples/burgers/config.py`, perhaps you could avoid duplicating all these lines by just having the single line `from examples.burgers.config import *`?